src/processing/scoutDataFormatter.ts

```diff
@@ -0,0 +1,58 @@
+import { addTeam, addScout } from '../state/actions'
+import { store } from '../state/store'
+import { ScoutSections } from '../data/scout'
+
+// Takes one object of raw scout data from one match. Will return a formatted object of the same data
+function formatScout(scoutData: object, gameName:string) {
+    const scoutMetrics = (scoutData as any)["metrics"]
+
+    // Initialize object to store the formatted scout data
+    var formattedMetrics: ScoutSections = {}
+
+    // Finds all the different sections in the scout data and stores them in an array
+    var allScoutSections: Array<string> = []
+    for (var metric in scoutMetrics) {
+        if (!allScoutSections.includes(scoutMetrics[metric]["category"])){
+            allScoutSections.push(scoutMetrics[metric]["category"])
+        }
+    }
+
+    // Add in the metrics object within each scout section
+    for (var section in allScoutSections){
```

Comment:
spaces between `)` and `{`
```suggestion
    for (var section in allScoutSections) {
```